web/modules/custom/dpl_cookies/dpl_cookies.module

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * DPL cookies module.
+ */
+
+/**
+ * Implements hook_preprocess_media().
+ *
+ * Adds cookie consent attributes to media video fields.
+ */
+function dpl_cookies_preprocess_field__media__video(array &$variables): void {
+  foreach ($variables['items'] as &$item) {
+    if (isset($item['content']['#attributes']['src'])) {
```

Comment:
Did not double check but if we can use Drupals field helpers I think we should do that.
IS it possible to do something like:
```php
if(!$item->get('content')->isEmpty()) && !$item['content']->get('src')->isEmpty()) {

Reply:
Maybe a version of this is possible, but then we first need to load the field as an entity object and then use the above if statement. 

I don't think this is viable here.